formatTest/unit_tests/expected_output/variants.re

```diff
@@ -269,9 +269,9 @@ let thisWontCompileButLetsSeeHowItFormats =
  * GADTs.
  */
 type term(_) =
-  | Int(int) : term(int)
+  | Int(int): term(int)
   | Add: term(((int, int) => int))
-  | App(term((('b) => 'a)), term('b)) : term('a);
+  | App(term((('b) => 'a)), term('b)): term('a);
```

Comment:
I notice that forcing the parenthesis around simple identifiers in lambdas (and their types) ends up making these nestings pretty paren-heavy. Compare:
```
 | App(term((('b) => 'a)), term('b)) : term('a);

 | App(term(('b => 'a)), term('b)) : term('a);
```

But this is probably partly because it looks like this has an additional parenthesis set in here. It could perhaps be as clean as:

```
 | App(term('b => 'a), term('b)) : term('a);
```